MSAL/test/unit/native_auth/controllers/MSALNativeAuthSignInControllerTests.swift

```diff
@@ -207,9 +207,8 @@ final class MSALNativeAuthSignInControllerTests: MSALNativeAuthTestCase {
         tokenRequestProviderMock.expectedUsername = expectedUsername
         tokenRequestProviderMock.expectedContext = expectedContext
 
-        let userAccountResult = MSALNativeAuthUserAccountResultStub.result
```

Comment:
This unit test was not working correctly and started failing after the change in the `SignInTestsValidatorHelper` 
The `userAccountResult` is never used, an error should be checked as per the unit test name.